packages/ckeditor5-ui/src/dialog/dialog.ts

```diff
@@ -467,9 +467,13 @@ export interface DialogDefinition {
 	 * Available dialog positions. By default `DialogViewPosition.EDITOR_CENTER` is used for {@link #isModal non-modals}
 	 * and `DialogViewPosition.SCREEN_CENTER` for modals.
 	 *
+	 * If set to a function, it will be called with the DOM root Rect and the dialog Rect as arguments.
```

Comment:
Followup after merge: "DOM root Rect" needs clarification. What it is? This is how the param was named inside the function in `DialogView` but we should propose more meaningful name and describe what it actually is, so somebody can correctly use it to position their dialog.